modules/network-flow-matrix.adoc

```diff
@@ -6,26 +6,91 @@
 
-To view or download the raw CSV content, see link:https://raw.githubusercontent.com/openshift/openshift-docs/main/snippets/network-flow-matrix.csv[this resource].
+* {product-title} on bare metal 
+* {sno} on bare metal
+* {product-title} on {aws-first}
+* {sno} on {aws-short}
 
-Additionally, consider the following dynamic port ranges when managing ingress traffic:
+Use the information in the appropriate network flow matrix to help you manage ingress traffic for your specific environment. You can restrict ingress traffic to essential flows to improve network security.
+
+Additionally, consider the following dynamic port ranges when managing ingress traffic for both bare metal and cloud environments:
 
 * `9000-9999`: Host level services
 * `30000-32767`: Kubernetes node ports
 * `49152-65535`: Dynamic or private ports
 
+To view or download the complete raw CSV content for an environment, see the following resources:
+
+//link:https://raw.githubusercontent.com/openshift/openshift-docs/main/snippets/network-flow-matrix.csv[this resource].
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-bm.csv[{product-title} on bare metal]
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-bm-sno.csv[{sno} on bare metal]
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-aws.csv[{product-title} on {aws-short}]
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-aws-sno.csv[{sno} on {aws-short}]
+
 [NOTE]
 ====
-The network flow matrix describes ingress traffic flows for a base {product-title} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrix does not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
+The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
 ====
 
-.Network flow matrix
+[id="network-flow-matrix-common_{context}"]
+== Network flow matrix base flows
+
+The following matrixes describe the ingress flows to {product-title} services that are common to the following environments:
+
+* {product-title} on bare metal 
+* {sno} on bare metal
+* {product-title} on {aws-first}
+* {sno} on {aws-short}
+
+[id="network-flow-matrix-control_{context}"]
+.Control plane node base flows
+[%header,format=csv]
+|===
+include::https://raw.githubusercontent.com/openshift-kni/commatrix/main/docs/4.18/commatrix-common-master.csv[]
```

Comment:
I think we should point to a specific branch and not main, release-4.18.
In addition, I'm updating 4.18 to stable to fit with OCP standard https://github.com/openshift-kni/commatrix/pull/73

Reply:
Done